.buildkite/ftr_configs.yml

```diff
@@ -49,6 +49,7 @@ disabled:
   - x-pack/plugins/apm/ftr_e2e/ftr_config.ts
   - x-pack/test_serverless/functional/test_suites/observability/cypress/config_headless.ts
   - x-pack/test_serverless/functional/test_suites/observability/cypress/config_runner.ts
+  - x-pack/test/security_solution_cypress/serverless_config.ts
```

Comment:
Nit: Let's move this up closer to other `x-pack/test/security_solution_cypress/` configs